### PR TITLE
Sea Devil Deep Glow

### PR DESCRIPTION
**Pull request by dfuchs2**
Added flavor text for Sea Devil's Deep Glow trait.

### CODE REVIEW

**Comment by dfuchs2** [comment]
https://github.com/bleehu/Compound_X/issues/781

**Review by bleehu — APPROVED** [review]
I'm good with it, but I'd like Rev to take a quick look before we shipit.

**Comment by rosebrallier** [comment]
This has been incorporated via PR #811. Recommend closing this PR pending merge of #811 @Reveraine @bleehu

**Comment by bleehu** [comment]
The two versions are different; the one in 811 has a reference to "light levels" which as far as I know aren't a thing yet?

**Review by Reveraine — APPROVED** [review]
Excellent!
Ship it!

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 1]
```diff
@@ -232,10 +232,10 @@ Deep Glow:
     Even those with low-light vision cannot see in total darkness without
 aid. A Sea Devil may concentrate on their bioluminescent scales, brightening
 them to a level that will allow others to see better in the darkness.
+
+   The Sea Devil has bioluminescent glands spread all over its body. When activated,
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: Gr: "their body" (people aren't "it"s)

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 2]
```diff
@@ -232,10 +232,10 @@ Deep Glow:
     Even those with low-light vision cannot see in total darkness without
 aid. A Sea Devil may concentrate on their bioluminescent scales, brightening
 them to a level that will allow others to see better in the darkness.
+
+   The Sea Devil has bioluminescent glands spread all over its body. When activated, 
+they cause it to glow a steady blue light. The light illuminates a 10m area around
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: Suggestion: "they glow with a steady blue light"

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 3]
```diff
@@ -232,10 +232,10 @@ Deep Glow:
     Even those with low-light vision cannot see in total darkness without
 aid. A Sea Devil may concentrate on their bioluminescent scales, brightening
 them to a level that will allow others to see better in the darkness.
+
+   The Sea Devil has bioluminescent glands spread all over its body. When activated, 
+they cause it to glow a steady blue light. The light illuminates a 10m area around 
+the Opaleite. The glow emitted from the Sea Devil makes it an easier target when
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: Gr: "them"

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 4]
```diff
@@ -232,10 +232,10 @@ Deep Glow:
...
+activated. Enemies recieve a +1 to hit.
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt]: Suggestion: "activated- enemies receive a +1 to hit them."
Sp: "receive"

**Review by rosebrallier — APPROVED** [review]
I like the progress we've made here!

**Pull request merged**